src/frontend/src/views/Project/Files/Console/index.tsx

```diff
@@ -45,6 +46,18 @@ export default class Xterm extends React.Component<ConsoleProps, ConsoleState> {
     return document.getElementsByClassName("xterm-rows")[0].childElementCount;
   }
 
+  getText(): string {
+        let rows = document.getElementsByClassName("xterm-rows")[0].children;

```

Comment:
Yikes, there's no way to do this through the API? Could this problem be solved more cleanly by associating the console with the project scope instead of the question scope?